chromadb/api/async_fastapi.py

```diff
@@ -103,7 +118,11 @@ def _get_client(self) -> httpx.AsyncClient:
             loop_hash = 0
 
         if loop_hash not in self._clients:
-            self._clients[loop_hash] = httpx.AsyncClient(timeout=None)
+            self._clients[loop_hash] = httpx.AsyncClient(
+                timeout=None, headers=self._headers, verify=self._verify
+            )
+        else:
+            self._clients[loop_hash].headers.update(self._headers)
```

Comment:
when would we need to update the headers like this?

Reply:
Tests were failing. I think it is due to having pytest use the same event loop

Reply:
but the clients aren't globally cached, the cache is local to the class instance--seems strange

Reply:
I agree, it took some time to figure out what the issue was. I can dig deeper to find why pytest does this.